systems/subscriber/sim-pool/test/integration/sim_pool_test.go

```diff
@@ -0,0 +1,119 @@
+package integration
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	confr "github.com/num30/config"
+	"github.com/ukama/ukama/systems/common/config"
+	"google.golang.org/grpc/credentials/insecure"
+
+	log "github.com/sirupsen/logrus"
+	"github.com/stretchr/testify/assert"
+	pb "github.com/ukama/ukama/systems/subscriber/sim-pool/pb/gen"
+	"google.golang.org/grpc"
+)
+
+type TestConfig struct {
+	ServiceHost string        `default:"localhost:9090"`
+	Queue       *config.Queue `default:"{}"`
+}
+
+var tConfig *TestConfig
+
+func init() {
+	tConfig = &TestConfig{}
+	r := confr.NewConfReader("integration")
+	r.Read(tConfig)
+
+	log.Info("Expected config ", "integration.yaml", " or env vars for ex: SERVICEHOST")
+	log.Infof("%+v", tConfig)
+}
+
+func Test_FullFlow(t *testing.T) {
+	const sysName = "sys"
```

Comment:
check these warings provided by github

Reply:
Am not seeing any warnings.Can you share a screenshot please?